src/components/Pizza.js

```diff
@@ -0,0 +1,299 @@
+import React, {useState,useEffect} from 'react';
+import {useHistory} from 'react-router-dom';
+import  * as Yup from 'yup';
+import axios from 'axios';
+
+function Pizza ({setPost}){
+    let history=useHistory();
+    const [formState, setFormState]=useState({
+        name:"",
+        size:"",
+        onion:false,
+        xtracheese:false,
+        mushroom:false,
+        pepperoni:false,
+        sausage:false,
+        ham:false,
+        bellpepper:false,
+        pineapple:false,
+        bacon:false,
+        olive:false,
+        chicken:false, 
+        special:"",
```

Comment:
I would suggest creating a constant e.g. `const initialFormState` so that you can reuse it in your `confirmOrder` function